package.json

```diff
@@ -14,8 +14,8 @@
     "fix:eslint": "yarn lint:eslint --fix",
     "fix:prettier": "yarn lint:prettier --write",
     "lint": "yarn lint:eslint && yarn lint:prettier",
```

Comment:
i found that eslint and prettier were not finding files when i was running it locally, remove the singlequotes and this works.